user/super/com/google/gwt/emul/java/lang/annotation/ElementType.java

```diff
@@ -22,5 +22,5 @@
  */
 public enum ElementType {
   ANNOTATION_TYPE, CONSTRUCTOR, FIELD, LOCAL_VARIABLE, METHOD, PACKAGE,
-  PARAMETER, TYPE, TYPE_PARAMETER, TYPE_USE,
+  PARAMETER, TYPE, TYPE_PARAMETER, TYPE_USE, MODULE,
```

Comment:
We don't seem to care about the exact ordinal so maybe keep them alphabetically sorted?

Reply:
Ah, I had intended to keep them in the original order (MODULE was the new last element as of Java 11).

I'll re-order just in case something silly happens with ordinal values... though as my trip into bytecode shows, the name of the enum constant is what matters for annotation values.

Reply:
On second thought, I do see that the entire list is alphabetical, my spot check of "hmm, TYPE_USE is last, so it must be append-only" was faulty. I'll stick with that ordering for now, and add RECORD_COMPONENT.